3.0/README.md

```diff
@@ -154,6 +154,55 @@ An array of tile endpoints. {z}, {x} and {y}, if present, are replaced with the
 ```
 
 ## 3.15 `vector_layers`
+
+REQUIRED. (if describing vector tiles)
+
+A JSON object whose value is an array of JSON objects. Each of those JSON objects describes one layer of vector tile data, and MUST contain the following key-value pairs:
+
+* `id` (string): The layer ID, which is referred to as the `name` of the layer in the [Mapbox Vector Tile spec](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#41-layers).
```

Comment:
It should be explicit that there is no requirement the id is unique, but some formats may prohibit overlapping id and zoom.

Reply:
Agreed, the enforcement of this is vague. 

cc @flippmoke @GretaCB